README.md

```diff
@@ -207,6 +213,17 @@ Start your extension from the Launcher.
 
 [![Launcher](launcher/preview.gif)](launcher)
 
+### [Leap Counter (Reusability 1C)](leap_counter_extension)
+
+Create your own reusable plugin components with Jupyter's "Provider-
+Consumer Pattern". This is one of three related extension examples
+that demonstrate JupyterLab's provider-consumer pattern, where plugins
+can depend on and reuse features from one another. The three packages
+that make up the complete example are "step_counter", "step_counter_extension",
+and "leap_counter_extension".
```

Comment:
Gathering this all into a single "Reusiability" point would reduce the repetition.

Reply:
Great idea, see previous comment! We should consolidate.